deployments/dev.pangeo.io/image/binder/environment.yml

```diff
@@ -1,62 +1,59 @@
 name: root
 channels:
-  - defaults
-  - pyviz/label/dev
-  - bokeh/label/dev
-  - intake
   - conda-forge
 dependencies:
-  - jupyterhub==0.9.2
-  - bokeh=0.12.16
+  - jupyterhub
+  - bokeh
   - bqplot
+  - click
   - cython
   - cytoolz
-  - dask=0.18.2
+  - dask
   - dask-ml
+  - dask-kubernetes
   - datashader
-  - distributed=1.22.1
+  - distributed
   - esmpy
   - fastparquet
-  - gcsfs=0.1.2
+  - fusepy
```

Comment:
do we still need fuse?

Reply:
@mrocklin, I thought that by using only conda-forge channel, we wouldn't need to do all that pinning. 
(I have no idea whether we need fusepy.  I was just repeating what was there already.)

Reply:
I suspect that the versions may still shift around in unpleasant ways.  Certainly having a consistent build chain helps to reduce conflicts due to different binary dependencies (like the GDAL pains of old) but versions of different libraries can still definitely shift away from each other.  Conda forge doesn't do any sort of ecosystem-wide testing.

Reply:
Okay, happy to "stand down" on this.  😸 
Would one strategy be to see if this works without pinning on "deploy", and if so, pin the versions on "staging"?

Reply:
Yeah, I think it's entirely reasonable to unpin everything, deploy to a less-important deployment verify that things work and if so, then re-pin everything to those new versions.  Presuambly we would do this every few months.

Reply:
Great!  So okay to merge?

Reply:
Assuming that you're ok passing through again and updating pinned versions then sure :)

Reply:
Yep!